### PR TITLE
isActive frontend

### PR DESCRIPTION
**Pull request by jstelnik**
**QUESTION**: should we change the view users page to include a dataframe similar to the categories, distributors page and also be able to add, edit, delete users there

### CODE REVIEW

**Comment by jstelnik** [comment]
QUESTION: should we change the view users page to include a dataframe similar to the categories, distributors page and also be able to add, edit, delete users there

**Comment by ChrisWilhelm** [comment]
@jstelnik no, not all employees have this ability. Just admins

File: client-employee/pages/14_Create_New_User.py [hunk 1]
```diff
@@ -24,11 +24,12 @@
         phoneNumber = left.text_input("Phone Number (Optional)", "")
         address = right.text_input("Address (Optional)", "")
         newSubmit = st.form_submit_button()
+        isActive = True
```
Comment (ChrisWilhelm) [inline client-employee/pages/14_Create_New_User.py:27]: the user connector does not get this field

**Review by ChrisWilhelm — CHANGES_REQUESTED** [review]

File: client-employee/routeConnectors/userConnector.py [hunk 2]
```diff
@@ -11,38 +11,43 @@ def getUsers():
...
   return r.data
 
-def postUser(email, name, phoneNumber=None, address=None ):
+def postUser(email, name, phoneNumber=None, address=None):
   if (phoneNumber is None and address is None ):
     f = json.dumps({
       "email": email,
-      "name": name
+      "name": name,
+      "isActive": True
```
Comment (ChrisWilhelm) [inline client-employee/routeConnectors/userConnector.py:19]: isActive is also not a field on the post request on the backend.

**Review by ChrisWilhelm — CHANGES_REQUESTED** [review]

**Comment by jstelnik** [comment]
> @jstelnik no, not all employees have this ability. Just admins

@ChrisWilhelm so if a user is logged in and they are an admin, they have the ability to change a user from active to inactive?

**Pull request closed without merging**